python/mujoco_mpc/demos/filter.py

```diff
@@ -12,87 +12,107 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 import matplotlib.pyplot as plt
 import mediapy as media
 import mujoco
-import numpy as np
-
 from mujoco_mpc import filter as filter_lib
+import numpy as np
```

Comment:
is this the correct [Google style](https://google.github.io/styleguide/pyguide.html#313-imports-formatting) import order?

Reply:
Yes. In this context mujoco_mpc and numpy are both third party libraries.